### PR TITLE
Added Gitpod FINALLY

### PR DESCRIPTION
**Pull request by LinuxGamer**
Added Gitpod badge for easy contribution without need of cloning and compiling (all through browser).

Fixes issues #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,30 @@ ownership of their social media presence.
...
+    src="https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod"
+    alt="Contribute with Gitpod"
+  />
+</a>
+
 To setup and install Shoutify for development, please read the [HACKING.md](.github/HACKING.md) document.
 
+<<<<<<< Updated upstream
 ### Contributing
+=======
```
Comment (TAKANOME-DEV) [inline README.md:48]: Please remove this line (`48`) to line `60` and also `46`. It's your stashed commit and it was deleted in the last PR.
Check the [README](https://github.com/TechSquidTV/Shoutify/blob/main/README.md) here

**Review by TAKANOME-DEV — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LinuxGamer) [reply README.md:48]: Ok, thanks I was wondering which bit was wrong XD

Comment (LinuxGamer) [reply README.md]: Fixed

**Review by TAKANOME-DEV — APPROVED** [review]

**Comment by KyleTryon** [comment]
Congrats on getting the PR working @LinuxGamer. I know there was a bit of a learning process involved.

I am still hesitant to merge this, not because of the work you have done but because of the service GitPod. I am not terribly familiar with them but this seems to act as an advertisement for an external commercial service, for which I believe there is an alternative already available here on GitHub (GitHub Code Spaces).

Is this something users need? 

Users will still need to configure Twitter credentials and create the .env file before the repo will work.
image


So the benefits I see from adding this are:

- Developers can run the shoutify dev server in the cloud
- Developers don't need NodeJS installed (because cloud)
- Developers can contribute from any device (again, cloud)

Versus, GitHub or GitHub Spaces:

- Users can contribute on github.com without running the repo
- Users who do want to build in the cloud may have their own preferred vendor, GH Code spaces, GitPod, or others. I don't have any personal reasons to push users toward one or the other.
- Is a cloud-ready dev space something developers actually need?

**Comment by KyleTryon** [comment]
Another vote from @TAKANOME-DEV to hold off on including this for now from the Discord discussion. Closing for now @LinuxGamer but we appreciate the contribution.

**Pull request closed without merging**